lib/config.js

```diff
@@ -6,15 +6,16 @@ import {config} from 'bedrock-web';
 // set bedrock-web-wallet default config
 config.wallet = {
   defaults: {
-    edvBaseUrl: `https://localhost:55443/edvs`,
+    onboardLink: 'https://localhost:38443/onboard',
+    edvBaseUrl: 'https://localhost:55443/edvs',
     // FIXME: do not use `referenceId` and put that information on the server,
     // rather have profile agent information store a location to read EDV IDs
     // from based on local names; remove this from the shared web app config
     edvs: {
       users: _getReferenceId('users'),
       credentials: _getReferenceId('credentials')
```

Comment:
This should match what the values were before (after running the old `_getReferenceId()` code. In a previous commit, these would have been erroneously set to: `localhost%3Ausers` and `localhost%3Acredentials`, respectively. We do not want to add `_getReferenceId()` back in, rather, we just wanted the proper static, default values here.

```suggestion
      users: 'localhost:users',
      credentials: 'localhost:credentials'
```